### PR TITLE
Sils/gitm

### PR DESCRIPTION
**Pull request by sils**

### CODE REVIEW

**Comment by sims1253** [comment]
ack 524d5c0984515c3d06688ffe65034138dca9619b

File: data/projects/implement_aspects.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+##### GSOC 2017 MIDTERM
+
+* The new aspect based configuration file works.
+* The new aspect configuration file is documented properly.
+
+##### GSOC 2017 FINAL
+
+* coala can properly visualize aspects.
+* coala can properly ignore aspects.
+* Everything is documented (and tested) properly.
```
Comment (sims1253) [inline data/projects/implement_aspects.md]: just to clarify. This is not about updating bears but making coala work with aspects? SO a different project from the  "Convert Bears to Aspects" project?

File: data/projects.js [hunk 2]
```diff
@@ -290,5 +290,21 @@ coala should support generating metrics for your code.",
 		"tags" : ["Core", "Bears", "Aspects"],
 		"markdown": "implement_aspects.md",
 		"collaborating_projects" : ["coala"]
+	},
+	{
+		"name" : "Implement GitLab Support for GitMate",
+		"desc" : "GitMate will support GitHub very soon with it's universal plugin \
+		system thing - GitLab is not far away thanks to your help!",
```
Comment (sims1253) [inline data/projects.js]: plugin system thing?

File: data/projects/gitmate_gitlab.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+GitMate is a plugin system for handling events from a Git hosting platform like
+GitHub. Eventually it will support automatic code review via coala, just as
+[our prototype](http://gitmate.io) does.
+
+We are currently developing it as a plugin platform so plugins can be reused
+for any git hoster and just plugged into hooks, similar to the GitHub webhooks.
+
+Task of this project is to:
```
Comment (sims1253) [inline data/projects/gitmate_gitlab.md]: The task...
or goal but proper sentence please :D

**Comment by sims1253** [comment]
ack c4bf916c8122966cd04d351d67819aa3c7bb5338

File: data/projects/implement_aspects.md (hunk 1 above)
Comment (sils) [inline data/projects/implement_aspects.md]: yes

File: data/projects.js (hunk 2 above)
Comment (sils) [inline data/projects.js]: yes. Nobody exactly knows yet.

**Comment by sils** [comment]
reack e01a5e8

**Comment by sims1253** [comment]
ack 9542099f32ce9237a928e6addc47756da9f9c363

File: data/projects.js (hunk 2 above)
Comment (sims1253) [inline data/projects.js]: remove the "thing"
it's just a plugin system

**Comment by sims1253** [comment]
ack 452b66b9337a3dab659fb4838beee3732d786cc9

**Comment by sims1253** [comment]
@rultor merge

**Pull request merged**